gpii/configs/preferencesServer.production.json

```diff
@@ -12,7 +12,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/preferencesServer/configs/development.json",
         "../node_modules/rawPreferencesServer/configs/production.json"
```

Comment:
We really need to end the practice of having a "production" config derive from a "development" one which stems from earlier, much lazier development practices. see https://issues.gpii.net/browse/GPII-1056